src/universal/modules/teamDashboard/components/ArchiveTeamConfirmation/ArchiveTeamConfirmation.js

```diff
@@ -0,0 +1,87 @@
+import React, {Component, PropTypes} from 'react';
+import {reduxForm, Field} from 'redux-form';
+import InputField from 'universal/components/InputField/InputField';
+import {cashay} from 'cashay';
+import Button from 'universal/components/Button/Button';
+import {css} from 'aphrodite-local-styles/no-important';
+import withStyles from 'universal/styles/withStyles';
+
+
+const styleThunk = () => ({
+  archiveTeamConfirmation: {
+    marginLeft: '66px'
+  }
+});
+
+@withStyles(styleThunk)
+@reduxForm({form: 'archiveTeamConfirmation'})
+
+export default class ArchiveTeamConfirmation extends Component {
+
+  static propTypes = {
+    teamName: PropTypes.string.isRequired,
+    teamId: PropTypes.string.isRequired,
+    handleSubmit: PropTypes.func.isRequired,
+    router: PropTypes.object.isRequired,
+    styles: PropTypes.object
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {showConfirmationField: false};
+  }
+
+  archiveTeam = () => {
+    const {teamName, teamId, router} = this.props;
+    const variables = {
+      updatedTeam: {
+        id: teamId,
+        name: teamName,
+        isArchived: true
+      }
+    };
+    cashay.mutate('archiveTeam', {variables});
+    router.push('/me');
+  }
+
+  formSubmit = (data) => {
+    const {teamName} = this.props;
+    const {archivedTeamName} = data;
+    if (teamName === archivedTeamName) {
```

Comment:
Bonus: reset the form when the name doesn't match.
Bonus bonus, make the form shake "no-no" using CSS animation keyframes :)

(feel free to ignore these suggestions)